vmc/resource_vmc_sddc.go

```diff
@@ -385,9 +387,66 @@ func resourceSddcDelete(d *schema.ResourceData, m interface{}) error {
 func resourceSddcUpdate(d *schema.ResourceData, m interface{}) error {
 	connector := (m.(*ConnectorWrapper)).Connector
 	esxsClient := sddcs.NewDefaultEsxsClient(connector)
+	sddcClient := orgs.NewDefaultSddcsClient(connector)
 	sddcID := d.Id()
 	orgID := (m.(*ConnectorWrapper)).OrgID
 
+	// Convert sddc from 1node to default
+	if d.HasChange("sddc_type") {
+		oldTmp, newTmp := d.GetChange("sddc_type")
+		oldType := oldTmp.(string)
+		newType := newTmp.(string)
+
+		// Validate for convert type params
+		if oldType == "1NODE" && (newType == "" || newType == "DEFAULT") {
+			oldTmp, newTmp := d.GetChange("num_host")
+			oldNum := oldTmp.(int)
+			newNum := newTmp.(int)
+
+			if oldNum != 1 {
+				return fmt.Errorf("Scale SDDC must from 1 host type")
+			} else if newNum == 2 { // 2node sddc creation
+				err := resourceSddcDelete(d, m)
+				if err != nil {
+					return err
+				}
+				return resourceSddcCreate(d, m)
```

Comment:
Have you tested if this part works ?

Reply:
Yes, it will create a new sddc after the older one has been deleted